src/site/content/en/blog/sms-otp-form/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: SMS OTP form best practices
+subhead: Learn how to optimize your SMS OTP form.
+authors:
+  - agektmr
+date: 2020-12-04
+hero: hero.jpg
+alt: A drawing of a woman using OTP to log in to a web app.
+
+description: |
+  Asking a user to provide an OTP (one-time password) delivered via SMS is a
+  common way to confirm a user's phone number. This post provides you with the
+  best practices to build an SMS OTP form for such use cases.
+
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - identity
+  - security
+  - forms
+  - otp
+---
+
+Asking a user to provide the OTP (one time password) delivered via SMS is a common way to confirm a user's phone number. There are a few use cases for SMS OTP:
+
+* **2 step verification**: In addition to username and password, SMS OTP can be
+used as a strong signal that the account is owned by the person who received the
+SMS OTP.
+* **Phone number verification**: Some services use a phone number as the user's
+primary identifier. In such services, users can enter their phone number and the
+OTP received via SMS to prove their identity. Sometimes it's combined with a PIN
+to constitute a 2 step verification.
+* **Account recovery**: When a user loses access to their account, there needs
+to be a way to recover it. Sending an email to their registered email address or
+an SMS OTP to their phone number are common account recovery methods.
+* **Payment confirmation**: In payment systems, some banks or credit card
+issuers request additional authentication from the payer for security reasons.
+SMS OTP is commonly used for that purpose.
+
+This post explains best practices to build an SMS OTP form for the above use
+cases.
+
+{% Aside 'caution' %}
+While this post discusses SMS OTP form best practices, be aware that SMS OTP is
+not the most secure method of authentication by itself. That's because phone
+numbers are known to be recycled and sometimes hijacked. And [the concept of OTP
+itself is not phishing resistant](https://youtu.be/kGGMgEfSzMw?t=1133).
+
+If you are looking for better security, consider using WebAuthn. You can learn
+more about it from the talk "[What's new in sign-up &
+sign-in](https://goo.gle/webauthn-video)" at the Chrome Dev Summit 2019.
```

Comment:
```suggestion
While this post discusses SMS OTP form best practices, be aware that SMS OTP is
not the most secure method of authentication by itself because phone
numbers can be recycled and sometimes hijacked. And [the concept of OTP
itself is not phishing resistant](https://youtu.be/kGGMgEfSzMw?t=1133).

If you are looking for better security, consider using [WebAuthn](https://webauthn.io/). Learn
more about it from the talk "[What's new in sign-up &
sign-in](https://goo.gle/webauthn-video)" at the Chrome Dev Summit 2019.
```